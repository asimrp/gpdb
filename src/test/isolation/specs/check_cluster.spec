session "s1"
step "down_segments" {select dbid, content, status, role, preferred_role, mode from gp_segment_configuration where status = 'd' or preferred_role != role;}

permutation "down_segments"
