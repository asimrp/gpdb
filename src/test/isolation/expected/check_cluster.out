Parsed test spec with 1 sessions

starting permutation: down_segments
step down_segments: select dbid, content, status, role, preferred_role, mode from gp_segment_configuration where status = 'd' or preferred_role != role;
dbid           content        status         role           preferred_role mode           

